### ISSUE TITLE
Cannot load save

### ISSUE THREAD

**Issue by kristianstroka** [caiiiycuk/js-dos.com#132]
Hi, I'm following your documentation, where you wrote ([https://js-dos.com/v7/build/docs/save-load](https://js-dos.com/v7/build/docs/save-load)) that we should use array as parameter for dos, but it is not actually working, when I use array, as first I have typescript error, that only string (not array) is allowed as first parameter of dos. And then when I use this way (in dos varaible I have dos instance and in saveBuffer I have int8array from persist function): 

`dos.run([my_bundle_url, URL.createObjectURL(new Blob([saveBuffer.buffer]))])`
then It doesnt work, it fails on wrong url of object .... instead of that I tried also this: 

`dos.run(my_bundle_url, URL.createObjectURL(new Blob([saveBuffer.buffer])))`

but also doesnt work...

then I use my server where I build api which gives me that int8array by url, so I dont use URL.createObject, but URL directly, so (in dos is my instance and in urlSave I have link to that array file):

`dos.run([my_bundle_url, urlSave])`
`dos.run(my_bundle_url, urlSave)`

non of them load saved stat ... what I'm doing wrong? I'm getting that int8array from function 
`const updated = await ci.persist()`

It mostly crashes with Unable to extract bundle archive or it run game without any save, from fresh ...

**Comment by caiiiycuk** [comment]
Hi. First of all it depends of which version you used. The documentation from link is for latest emulators and emulators-ui packages, it still not deployed as js-dos package. Plesase confirm that you used last emulatros+emulatros-ui packages.

**Comment by caiiiycuk** [comment]
This snipped how I used it internally in tests:
```ts
    let cachedBundle: Uint8Array = new Uint8Array();
    test(name + " should store fs updates between sessions [empty db]", async () => {
        const buffer = await HTTPRequest("digger.jsdos", {
            cache: new CacheNoop(),
            responseType: "arraybuffer",
        });

        const ci = await factory(new Uint8Array(buffer as ArrayBuffer));
        assert.ok(ci);
        cachedBundle = await ci.persist();
        assert.ok(cachedBundle, "cachedBundle is undefined");
        await compareAndExit("persistent-mount.png", ci);
    });

    test(name + " should store fs updates between sessions [existent db]", async () => {
        const buffer = await HTTPRequest("digger.jsdos", {
            cache: new CacheNoop(),
            responseType: "arraybuffer",
        });

        const ci = await factory([new Uint8Array(buffer as ArrayBuffer), cachedBundle]);
        assert.ok(ci);
        cachedBundle = new Uint8Array();
        await compareAndExit("persistent-mount-second.png", ci);
    });
```

**Comment by caiiiycuk** [comment]
But for js-dos API it's correct to use urls, like this:
```
dos.run(url, chagnesUrl);
```

**Comment by caiiiycuk** [comment]
Just pushed `7.0.0-beta.r30` to npm, please try is it works for you

**Comment by kristianstroka** [comment]
it is still not working for me, now I have array (not Uint8Array, just simple array) of all values from persist state in variable: 

`let saveData = [lotofnumbers]`

I tried to load this save data as follow: 

`dos.run(bundleUrl, saveData)`

jsdos runs as it should, but data are not saved, behaviour is like they are not updated, but I'm sure that they has been updated. 

What can be wrong there? Should I transform saveData array to something else?
